doc/cookbook/curl-mock/CurlMock.lhs

```diff
@@ -86,7 +86,7 @@ listFromAPI :: (HasForeign lang ftype api, GenerateList ftype (Foreign ftype api
 ```
 
 This looks a bit confusing...
-[Here](https://hackage.haskell.org/package/servant-foreign-0.11.1/docs/Servant-Foreign.html#t:HasForeignType) is the documentation for the `HasForeign` typeclass.
+[Here](https://hackage.haskell.org/package/servant-foreign/docs/Servant-Foreign.html#t:HasForeignType) is the documentation for the `HasForeign` typeclass.
```

Comment:
Changed all the cookbook links to point to latest version of the package on hackage. I checked that all the links I changed are alive.

Reply:
Should this be https://hackage.haskell.org/package/servant-foreign-0.15.1/docs/Servant-Foreign.html#t:HasForeign instead?

Reply:
You're right. I was confused, because the original code lined to HasForeignType, which also exists in the package. But from the context it's clear that link to HasForeign is needed here. Fixed.